tf2/include/tf2/LinearMath/MinMax.h

```diff
@@ -18,25 +18,25 @@ subject to the following restrictions:
 #define GEN_MINMAX_H
 
 template <class T>
-TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
+TF2_PUBLIC TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
```

Comment:
I was reading about exporting and inline here:

https://msdn.microsoft.com/en-us/library/xa0d9ste.aspx

So they *can* be exported, but it's not clear to me whether they should or shouldn't be.